frontend/src/templates/Field/Checkbox/CheckboxField.tsx

```diff
@@ -18,19 +18,22 @@ import FormErrorMessage from '~components/FormControl/FormErrorMessage'
 import { BaseFieldProps, FieldContainer } from '../FieldContainer'
 import { CheckboxFieldInputs, CheckboxFieldSchema } from '../types'
 
-export const CHECKBOX_OTHERS_INPUT_KEY = 'othersInput'
-export const CHECKBOX_OTHERS_INPUT_VALUE =
-  '!!FORMSG_INTERNAL_CHECKBOX_OTHERS_VALUE!!'
+import {
+  CHECKBOX_OTHERS_INPUT_KEY,
+  CHECKBOX_OTHERS_INPUT_VALUE,
+} from './constants'
 
 export interface CheckboxFieldProps extends BaseFieldProps {
   schema: CheckboxFieldSchema
+  disableRequiredValidation?: boolean
```

Comment:
`disableRequiredValidation` is currently used to bypass validation since MRF fields can be left empty for subsequent respondents to fill.

How do we intend to implement response validation in subsequent stages of MRF?

Do we intend to enforce it at the respondent level? e.g. field 1, 2 , 3 is required for respondent 1/submission 1.